json-endpoints/class.wpcom-json-api-bulk-delete-post-endpoint.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class WPCOM_JSON_API_Bulk_Delete_Post_Endpoint extends WPCOM_JSON_API_Update_Post_v1_1_Endpoint {
+	function callback( $path = '', $blog_id = 0 ) {
+		$blog_id = $this->api->switch_to_blog_and_validate_user( $this->api->get_blog_id( $blog_id ) );
+		if ( is_wp_error( $blog_id ) ) {
+			return $blog_id;
+		}
+
+		$input = $this->input();
+
+		$post_ids = (array) $input['post_ids'];
```

Comment:
Should we return `WP_Error` here with some helpful message in case `$post_ids` is empty?